api/Window.json

```diff
@@ -972,10 +972,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/cancelAnimationFrame",
```

Comment:
Can you test for when the prefixed variants were added? They're included for `requestAnimationFrame`: https://developer.mozilla.org/en-US/docs/Web/API/Window/requestAnimationFrame

Tests:
http://mdn-bcd-collector.appspot.com/tests/api/Window/webkitCancelAnimationFrame
http://mdn-bcd-collector.appspot.com/tests/api/Window/webkitCancelRequestAnimationFrame (after https://github.com/foolip/mdn-bcd-collector/pull/601)